_data/vangelists/vangelists.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Vangelis Tsiatsianas
+institution: Computer Science Department - University of Crete
+profile_pic: vangelists.png
+quote: '"'
```

Comment:
We were expecting some more content here 🤔

Reply:
Could not find a quote! 😀

Reply:
Could you use the literal quote as it is or is a phrase absolutely required? 🙂

Reply:
Yep, it is a requirement

Reply:
Updated! 🙂